apple/RNCWebView.m

```diff
@@ -324,6 +325,10 @@ - (void)removeFromSuperview
 -(void)showFullScreenVideoStatusBars
 {
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
+    if (!_autoManageStatusBarEnabled) {
+      return;
+    }
```

Comment:
I believe we need this guard in `hideFullScreenVIdeoStatusBars` (next method down) too.

Reply:
done